app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  before_action :restrict_access, only: %i[show]
+
+  def show
+    @events = Event.where(creator_id: current_user.id)
+    @attendees = Attendance.where(user_id: current_user.id)
+  end
```

Comment:
You could use a `current_user` that has the user instance for the logged in user and then load all created events by this user as: 

```
created_events = current_user.events
past_events = current_user.attendances.past
upcoming_events = current_user.attendances.upcoming
```

[`REQUIRED`]

References:

https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png

Reply:
we load past and ucoming events in each variable